plugins/DigitalLibrary/src/DigitalFactoryController.py

```diff
@@ -166,6 +170,11 @@ def userAccountHasLibraryAccess(self) -> bool:
             subscriptions = self._account.userProfile.get("subscriptions", [])
             if len(subscriptions) > 0:
                 return True
+        if self._user_has_access:
+            # The user has access even though they have no subscriptions. This means they are an Essential user and they
+            # have limited personal private projects available. In this case, we need to check whether they have already
+            # reached their limit.
+            self._api.checkUserCanCreateNewLibraryProject(callback = self.setCanCreateNewLibraryProject)
```

Comment:
Shouldn't we just always check if a user has a limit? That way if it ever changes, Cura is ready for that.

Reply:
Indeed, there was no need to check for the total number of subscriptions any more, since the variable `_user_has_access` already has covered the case the user has a subscription. I removed the number of subscriptions check in https://github.com/Ultimaker/Cura/pull/10131/commits/3dbbe7276baa3b201698e19bfbacbef45a3d8464.